components/libc/compilers/common/ctime.c

```diff
@@ -761,30 +755,25 @@ int clock_nanosleep(clockid_t clockid, int flags, const struct timespec *rqtp, s
     case CLOCK_MONOTONIC:
     case CLOCK_CPUTIME_ID:
     {
-        uint64_t cpu_tick, cpu_tick_old;
-        cpu_tick_old = clock_cpu_gettime();
-        rt_tick_t tick;
+        uint64_t cpu_tick_old = clock_cpu_gettime();
```

Comment:
除非POSIX或者ISO-ANSI兼容，建议用rt_uint64_t 其会自动匹配libc的数据类型或使用自己独立的类型
```suggestion
        rt_uint64_t cpu_tick_old = clock_cpu_gettime();
```